aten/src/THC/generic/THCTensorMathMagma.cu

```diff
@@ -140,8 +140,10 @@ THC_API void THCTensor_(gels)(THCState *state, THCTensor *rb_, THCTensor *ra_, T
 #ifdef USE_MAGMA
   THArgCheck(a_->nDimension == 2, 1, "A should be 2 dimensional");
   THArgCheck(b_->nDimension == 2, 1, "b should be 2 dimensional");
-  THArgCheck(a_->size[0] == b_->size[0], 2, "size incompatible A,b");
-  THArgCheck(a_->size[0] >= a_->size[1], 2, "A should have m >= n");
+  THArgCheck(a_->size[0] == b_->size[0], 2, "Expected A and b to have same size "
+      "at dim 0, but they have incompatible sizes");
```

Comment:
nit: tell the user the sizes

Reply:
THArgCheck doesn't support this unfortunately. I can switch to THError but I don't think it is very necessary.